### DISCUSSION TITLE
Everyone can see the ticket that was created.

### DISCUSSION THREAD

**Discussion by Xalthorak** [LunarTaku/djs-ticket-system#7] [Error] [answered]
When a user creates a ticket, everyone sees the ticket that was created.
![image](https://user-images.githubusercontent.com/80824386/185219220-03f6faaa-4857-4625-a615-7f6bbbf5be3c.png)
![image](https://user-images.githubusercontent.com/80824386/185219325-bb2bdee0-337a-44e4-90a2-6b1b7b83568d.png)

**Comment by LunarTaku** [comment, marked as answer]
i have fixed this in the new release, please check that out since it fixed ALOT of problems and adds new functions!
